File: src/CalculadoraConsole.py
from CalculadoraAbstrata import CalculadoraAbstrata


class CalculadoraConsole(CalculadoraAbstrata):
    entrada = ""
    resultado = 0
    
    def main(self):
        print("Bem vindo a calculadora!")
        print("")
        print("Digite a operação")
        self.entrada = input("")
        self.resultado = self.calcula(self.entrada)
        print(self.resultado)


if __name__ == "__main__":
    c = CalculadoraConsole()
    c.main()

File: java/calculadora/src/main/java/com/exemplo/CalculadoraConsole.java
package com.exemplo;

import java.util.Scanner;

public class CalculadoraConsole extends CalculadoraAbstrata {

    public void main() {
        Scanner scanner = new Scanner(System.in);

        System.out.println("Bem vindo a calculadora!");
        System.out.println("Insira a conta:");

        String entrada = scanner.nextLine();
        double saida = calcula(entrada);

        System.out.println(saida);
    }
}
